src/js/components/award/AwardAmounts.jsx

```diff
@@ -50,21 +51,41 @@ export default class AwardAmounts extends React.Component {
         const recipient = this.props.selectedAward.recipient_name.toLowerCase();
 
         const ceiling =
-            accounting.unformat(this.props.selectedAward.potential_total_value_of_award);
-        const current = accounting.unformat(this.props.selectedAward.total_obligation);
+            MoneyFormatter.formatMoneyWithPrecision(
+                this.props.selectedAward.potential_total_value_of_award, 0);
+        const current = MoneyFormatter.formatMoneyWithPrecision(
+            this.props.selectedAward.total_obligation, 0);
 
         // calculate the percentage spent
         let percentage = 'N/A';
         if (ceiling && ceiling !== 0) {
             percentage = Math.floor((current / ceiling) * 1000) / 10;
```

Comment:
You can't do this any more because you've converted `current` and `ceiling` to currency-formatted strings. You need to perform the percentage calculating using real numbers.

This breaks the contract page (award 69626):
> This contract was awarded to Carahsoft Technology Corporation with a ceiling of  $2,856.  Of this amount, NaN% ($2,856) has been obligated.